components/data-services/org.wso2.carbon.dataservices.core/src/main/java/org/wso2/carbon/dataservices/core/engine/CallQuery.java

```diff
@@ -315,7 +315,13 @@ protected void executeElement(XMLStreamWriter xmlWriter, ExternalParamCollection
 				paramObj = params.getParam(paramName);
 			}
 			if (paramObj != null) {
-			    qparams.put(withParam.getName(), paramObj.getValue());
+				if(paramObj.getValue().getScalarValue() ==  null ||(paramObj.getValue().getScalarValue().equals("") &&
```

Comment:
Shall we fix the formatting issues here?

Reply:
Fixed